specification/web/resource-manager/Microsoft.Web/stable/2020-12-01/Global.json

```diff
@@ -0,0 +1,329 @@
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/deletedSites/{deletedSiteId}": {
+      "get": {
+        "tags": [
+          "Global"
+        ],
+        "summary": "Get deleted app for a subscription.",
+        "description": "Description for Get deleted app for a subscription.",
+        "operationId": "Global_GetDeletedWebApp",
+        "parameters": [
+          {
+            "name": "deletedSiteId",
+            "in": "path",
+            "description": "The numeric ID of the deleted app, e.g. 12345",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DeletedSite"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Deleted Web App": {
+            "$ref": "./examples/GetDeletedWebApp.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/deletedSites/{deletedSiteId}/snapshots": {
+      "get": {
+        "tags": [
+          "Global"
+        ],
+        "summary": "Get all deleted apps for a subscription.",
+        "description": "Description for Get all deleted apps for a subscription.",
+        "operationId": "Global_GetDeletedWebAppSnapshots",
+        "parameters": [
+          {
+            "name": "deletedSiteId",
+            "in": "path",
+            "description": "The numeric ID of the deleted app, e.g. 12345",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "./CommonDefinitions.json#/definitions/Snapshot"
+              }
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Deleted Web App Snapshots": {
+            "$ref": "./examples/GetDeletedWebAppSnapshots.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/locations/{location}/operationResults": {
```

Comment:
>/subscriptions/{subscriptionId}/providers/Microsoft.Web/locations/{location}/operationResults":  [](start = 5, length = 96)

Do you really need an api to list all async operations? Usually the async pattern is that the client just need to call the uri in the location or azure-asyncOperation header to track the async operation.
[https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#creating-or-updating-resources-asynchronously](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#creating-or-updating-resources-asynchronously)

Reply:
removed from swagger